src/common/function/src/system/build.rs

```diff
@@ -52,20 +52,23 @@ impl Function for BuildFunction {
     }
 
     fn eval(&self, _func_ctx: FunctionContext, _columns: &[VectorRef]) -> Result<VectorRef> {
-        let build_info = format!(
-            "branch: {}\ncommit: {}\ncommit short: {}\ndirty: {}\nversion: {}",
-            build_data::get_git_branch().unwrap_or_else(|_| DEFAULT_VALUE.to_string()),
-            build_data::get_git_commit().unwrap_or_else(|_| DEFAULT_VALUE.to_string()),
-            build_data::get_git_commit_short().unwrap_or_else(|_| DEFAULT_VALUE.to_string()),
-            build_data::get_git_dirty().map_or(DEFAULT_VALUE.to_string(), |v| v.to_string()),
-            env!("CARGO_PKG_VERSION")
-        );
-
+        let build_info = make_build_info();
         let v = Arc::new(StringVector::from(vec![build_info]));
         Ok(v)
     }
 }
 
+fn make_build_info() -> String {
+    format!(
+        "branch: {}\ncommit: {}\ncommit short: {}\ndirty: {}\nversion: {}",
+        build_data::get_git_branch().unwrap_or_else(|_| DEFAULT_VALUE.to_string()),
+        build_data::get_git_commit().unwrap_or_else(|_| DEFAULT_VALUE.to_string()),
+        build_data::get_git_commit_short().unwrap_or_else(|_| DEFAULT_VALUE.to_string()),
+        build_data::get_git_dirty().map_or(DEFAULT_VALUE.to_string(), |v| v.to_string()),
+        env!("CARGO_PKG_VERSION")
+    )
```

Comment:
This is similar to `setup_git_versions`. Is it possible to refactor them to based on one common method?

Reply:
Sounds good. Let me push a commit for this (maybe in hours, I have a few other tasks to follow now)

Reply:
Fixed.